### PR TITLE
Changed the user location symbol.

### PR DESCRIPTION
**Pull request by teamagam3**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/83" target="_blank">ReviewNinja

File: app/src/main/assets/GimelGimel/VectorLayer.js [hunk 1]
```diff
@@ -216,6 +211,49 @@ var symbolToBillboardSymbol = function (symbol) {
...
+        throw new Error("Given symbol argument is missing data or of unsupported type");
+    }
+
+    return entity;
+};
+
+var createPoint = function() {
+    return {
+        pixelSize : 5,
+        color : Cesium.Color.RED,
```
Comment (teamagam1) [inline app/src/main/assets/GimelGimel/VectorLayer.js]: not always RED

Comment (teamagam1) [inline app/src/main/assets/GimelGimel/VectorLayer.js]: !fix

File: app/src/main/assets/GimelGimel/VectorLayer.js [hunk 2]
```diff
@@ -216,6 +211,49 @@ var symbolToBillboardSymbol = function (symbol) {
     }
 };
 
+var createEntityFromSymbol = function (id, location, symbol) {
```
Comment (BarAgam) [inline app/src/main/assets/GimelGimel/VectorLayer.js]: this method creates entity from both symbol and location, and not symbol alone. !fix

Comment (teamagam3) [inline app/src/main/assets/GimelGimel/VectorLayer.js]: !fixed

File: app/src/main/assets/GimelGimel/VectorLayer.js (hunk 1 above)
Comment (teamagam3) [inline app/src/main/assets/GimelGimel/VectorLayer.js]: !fixed

**Pull request merged**